src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,252 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Random;
+import java.io.FileWriter;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.IOException;
+import java.io.File;
+import duke.task.Task;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
```

Comment:
Good job with these import statements that specify explicitly, the classes that are to be maintained here.